### PR TITLE
AzureRT PR S50

### PR DESCRIPTION
**Pull request by huangpf**
- Format Image Cmdlet Output
- Overwrite the 'Clean-ResourceGroup' function

### CODE REVIEW

**Comment by AzureRT** [comment]
retest this please

**Comment by huangpf** [comment]
retest this please

**Comment by huangpf** [comment]
retest this please

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1 [hunk 1]
```diff
@@ -284,9 +293,37 @@ function Get-DefaultVMConfig
...
+
+# Assert Output Contains
+function Assert-OutputContains
+{
+    param([string] $cmd, [string[]] $sstr)
+    
+    $st = Write-Verbose ('Running Command : ' + $cmd);
+    $output = Invoke-Expression $cmd | Out-String;
+
+    $max_output_len = 1500;
```
Comment (hyonholee) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1:311]: Is this just an arbitrary number?  Can you add comment about why you choose this number for max_output_len?

Comment (huangpf) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1:311]: This part is for logging, and it doesn't help if the log is too long.

**Comment by huangpf** [comment]
retest this please

**Pull request merged**